diagnostic.js

```diff
@@ -41,7 +45,7 @@ db.dropDatabase();
 // House Arryn is not honorable!  Delete their motto.
 
 // your code begins here
-
+db.houses.deleteOne({motto:"As High as Honor" })
```

Comment:
Nope. `deleteOne` deletes the entire object. What you are trying to do is an update.

```js
db.houses.update({ name: "House Arryn"}, { name: "House Arryn"});
```

OR

```js
db.houses.update({ name: "House Arryn"}, { $unset: { motto: 'anything'} });
```